packages/openstackservice/buildinfo.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Why do we need a default variant for this package if it's only used in the Windows build?

Also, what does this package do?

Reply:
it did not build without default versions.
This package is to loosely replicate the systemd service manager as best as we can in windows. It runs a binary as a service and grabs the output and puts that in a log file.